### ISSUE TITLE
Support for inspecting the change queue.

### ISSUE THREAD

**Issue by dlemmermann** [TomasMikula/UndoFX#17]
It would be nice if changes could be named so that an undo / redo UI could display the name of the last change that will be undone or redone, e.g. "Undo 'Move Object'".

**Comment by TomasMikula** [comment]
I suppose that rather than any support for naming changes, we need a way to iterate over the changes in the queue. How one then extracts a name from a change is up to the user and the type of the change.

**Comment by dlemmermann** [comment]
That would be good, too. But for UI purposes it would be more convenient if
a button or menu item could bind itself to a property.
Tomas Mikula <notifications@github.com> schrieb am Mi. 26. Apr. 2017 um
04:45:

**Comment by TomasMikula** [comment]
If you only want to display a single item to undo/redo, then a property makes sense 👍 

Something like

```java
public Val<C> nextToUndoProperty();
public Val<C> nextToRedoProperty();
```

**Comment by dlemmermann** [comment]
Yes, something like that.

**Comment by TomasMikula** [comment]
Available in release 1.3.0.

**Comment by dlemmermann** [comment]
Awesome, thanks!

**Comment by dlemmermann** [comment]
I think your commit failed. I can't see the new properties (nextToUndo(), nextToRedo()) in 1.3. Also, what is the best way to bind a boolean property to the new Val<Boolean> return types of undo/redoAvailableProperty()?

**Comment by TomasMikula** [comment]
Ah, you are right, silly me! I implemented them but forgot to put them on public API.

> what is the best way to bind a boolean property to the new Val return types

Just plain old `bind` should work, as [in the demo](https://github.com/TomasMikula/UndoFX/blob/f32c6cbc09053f5500dca94354228f9900480889/undofx-demos/src/main/java/org/fxmisc/undo/demo/CircleProperties.java#L196-L197).

**Comment by TomasMikula** [comment]
Should be all fixed in [1.3.1](https://github.com/TomasMikula/UndoFX/releases/tag/v1.3.1).
